assets/blocks/quiz/question-block/settings/question-grade-settings.js

```diff
@@ -0,0 +1,49 @@
+import { BaseControl, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Question block grade settings.
+ *
+ * @param {Object}   props
+ * @param {Object}   props.options
+ * @param {number}   props.options.grade Question grade.
+ * @param {Function} props.setOptions
+ * @param {string}   props.clientId
+ */
+const QuestionGradeSettings = ( {
+	options: { grade = 1 },
+	setOptions,
+	clientId,
+} ) => {
+	const id = `sensei-lms-question-block__grade-control-${ clientId }`;
+	return (
+		<BaseControl
+			id={ id }
+			label={ __( 'Grade', 'sensei-lms' ) }
+			className="sensei-lms-question-block__grade-control"
+		>
+			<div className="sensei-lms-question-block__grade-control__controls">
+				<input
```

Comment:
Just to comment here that as soon one of these PRs is merged, we could refactor this part to use the same component created here: https://github.com/Automattic/sensei/pull/3987/files#diff-14065d250f2a91f20b02fe50b9fe4eb663abe18cd1c1746b2c939a3b4527de9c

I think it'll just need a small tweak adding a prop to a custom reset value.

Reply:
Updated to use `NumberControl` — cd24e5d, 448ab8e :keycap_ten: